aspnetcore/data/ef-rp/intro/samples/StageSnapShots/cu-part3-sorting/appsettings.json

```diff
@@ -0,0 +1,11 @@
+﻿{
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=CU-3s;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
It's usually best to add "ConnectRetryCount=0" to connection strings used with EF to stop SQLClient from hanging inappropriately.